File: Sistema para Bibliotecas Públicas/my-app/src/models/Leitor.ts
import { Emprestimo } from "./Emprestimo";

export interface Leitor{
    leitorId?: string;
    emprestimoId?: string,
    nome: string;
    sobrenome: string;
    telefone: string;
    email: string;
    cpf: string;
    emprestimo?: Emprestimo;
}

File: Sistema para Bibliotecas Públicas/Gerenciador Biblioteca/Biblioteca/Models/Leitor.cs
namespace Biblioteca.Models
{
    public class Leitor
    {
        public Leitor()
        {
            LeitorId = Guid.NewGuid().ToString();
        }

        public string? LeitorId { get; set;}

        public string? Nome { get; set; }

        public string? Sobrenome { get; set; }

        public string? Email { get; set;}

        public string Telefone { get; set; }

        public string CPF { get; set; }
    }
}